transport/src/test/java/io/zeebe/transport/impl/AtomixTransportTest.java

```diff
@@ -202,7 +205,7 @@ public void shouldFailResponseWhenRequestHandlerThrowsException() {
     // when
     final var requestFuture =
         clientTransport.sendRequestWithRetry(
-            nodeAddressSupplier, new Request("messageABC"), Duration.ofSeconds(1));
+            nodeAddressSupplier, new Request("messageABC"), RETRY_TIMEOUT);
```

Comment:
It is a bit confusing why this is called retry timeout for the request timeout.

Reply:
Good point.